docs/readme.md

```diff
@@ -1 +1,22 @@
-- placeholder -
+## Plugin Install Instructions
+
+
+### Supported IDEs
+
+
+The Flutter Plugin can be installed on following IntelliJ-based products:
+
+* IntelliJ 2016.2+ ([Ultimate or Community](https://www.jetbrains.com/idea/download/))
+
+
+### Installation
```

Comment:
We seem to be missing something. When I tested these in a clean VM, I got the following error:

```
Problems found loading plugins: Plugin "Flutter" was not loaded: required plugin "Dart" not installed.
```